R/S3.R

```diff
@@ -1805,3 +1805,31 @@ pander.nls <- function(x, digits = panderOptions('digits'), show.convergence = F
     pander.summary.nls(summary(x), summary = FALSE, add.significance.stars = FALSE,
                        digits = digits, show.convergence = show.convergence, ...)
 }
+
+#' Prints an arima object from stats package in Pandoc's markdown.
+#' @param x an arima object
+#' @param digits number of digits of precision
+#' @param se if to include standard error in coefficients table (default \code{TRUE})
+#' @param ... optional parameters passed to raw \code{pandoc.table} function
+#' @export
+pander.Arima <- function(x, digits = panderOptions('digits'), se = TRUE,...) {
+    cat('\nCall:', pandoc.formula.return(x$call), '', sep = '\n')
+    cn <- names(x$coef)
+    coef <- matrix(x$coef, nrow = 1)
+    colnames(coef) <- cn
+    if (se && nrow(x$var.coef)) {
```

Comment:
I see the `diag` in the lines below, but here you are doing a check on `nrow` without comparing the results of that with any integer -- and I though that the returned integer would be interpreted as `TRUE` only if it's `1L`. Since then, I've done my homework and it seems that any non-zero number is interpreted as `TRUE` -- so no problem here, sorry for the trouble.